getting_started/administrators.adoc

```diff
@@ -166,13 +166,6 @@ would generate their own keys and not have access to the system keys.
 Now that you have {product-title} successfully running in your environment,
 xref:try-it-out[try it out] by walking through a sample application lifecycle.
 
-=== Method 3: Building from Source [[building-from-source]]
-You can build {product-title} from source locally or using
-https://www.vagrantup.com/[Vagrant]. See the {product-title} repository on
-GitHub
-https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc#develop-on-virtual-machine-using-vagrant[for
-instructions].
-
```

Comment:
Do you need to remove the " If you wish to run the server from a Windows or Mac OS X host, you should download and run the Origin Vagrant image as described in xref:building-from-source[Method 3]." from the prerequisites section?